modules/_node-scripts/package.json

```diff
@@ -5,14 +5,12 @@
 	"dependencies": {
 		"@typescript-eslint/parser": "^7.8.0",
 		"babel-eslint": "^10.1.0",
-		"cross-env": "^7.0.3",
-		"esbuild": "^0.20.2",
+		"esbuild": "0.20.2",
```

Comment:
👍  we shoudl probably lock all versions in this package.json. I'll add to my list...